corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -324,7 +324,7 @@ hqDefine('cloudcare/js/utils', [
 
         $el.on("focusout", $el.data("DateTimePicker").hide);
         $el.attr("placeholder", dateFormat);
-        $el.attr("pattern", "[0-9-/]+");
+        $el.attr("pattern", "[0-9\-\/]+");
```

Comment:
Please use `eslint-disable-line` (possibly with an explanation) so someone doesn't innocently undo this.